web/server/api/endpoints/models.py

```diff
@@ -23,16 +24,28 @@ def get_models(
     context: Context = Depends(get_loaded_context),
 ) -> t.List[models.Model]:
     """Get a mapping of model names to model metadata"""
+    try:
+        context.load()
+    except Exception:
+        raise ApiException(
+            message="Unable to get models",
+            origin="API -> models -> get_models",
+        )
+
+    return get_all_models(context)
+
+
+def get_all_models(context: Context = Depends(get_loaded_context)) -> t.List[models.Model]:
```

Comment:
You can remove `Depends()`.
```suggestion
def get_all_models(context: Context) -> t.List[models.Model]:
```